content/_index.md

```diff
@@ -53,10 +53,19 @@ You may also use the permalinks given above to reference from your project home
 
 <strong class="important">Important!</strong> You must add a contact method to the placeholder in the document so that people know how to report violations.
 
-If you are using a markdown README file in your source code repository, you may want to add a badge like this one ![Contributor Covenant Badge](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg) using the code below.
+If you are using a README file in your source code repository, you may want to add a badge like this one ![Contributor Covenant Badge](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg) using the code below.
+
+For a markdown README, add the below code where you would like to insert the badge:
 ```
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
 ```
+For a reStructuredText README, place `|Contributor Covenant|` where you would like the badge and add the below to your README file:
+```
+
+.. |Contributor Covenant| image:: https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg
+   :target: code_of_conduct.md
```

Comment:
Should this have a different extension?

Reply:
It could, but I don't think it has to - I still use the md file in my repo, which works fine within GitHub. The issue it causes for automatic generation of documentation with Read the Docs and Sphinx is related to the url itself (it being relative within GitHub), not the file type.